sdk/storage/azure-storage-blob/azure/storage/blob/_shared/shared_access_signature.py

```diff
@@ -225,6 +229,18 @@ def add_override_response_headers(
         self._add_query(QueryStringConstants.SIGNED_CONTENT_LANGUAGE, content_language)
         self._add_query(QueryStringConstants.SIGNED_CONTENT_TYPE, content_type)
 
+    def add_request_headers(self, request_headers):
+        if not request_headers:
+            return
+        serialized = [str(k) + ":" + str(v) for k, v in request_headers.items()]
```

Comment:
nit: This is already a `Dict[str, str]` right? So, you shouldn't need the `str` on the key and value.